### DISCUSSION TITLE
Install package in pytorch environments

### DISCUSSION THREAD

**Discussion by yokokei** [PaddlePaddle/PaddleSpatial#206] [BaiduKDDCup2022] [answered]
Hi, could you install the following libraries in the Pytorch environment?
- torch-geometric-temporal (if [#204 ](https://github.com/PaddlePaddle/PaddleSpatial/discussions/204)works)
- tsfresh

Thank you!

**Comment by ramber1836** [comment]
Sorry, torch-geometric-temporal cannot be installed as discussed in https://github.com/PaddlePaddle/PaddleSpatial/discussions/78 .

**Reply by yokokei** [reply]
Sure. Thank you for your help.
Could you install tsfresh?

**Comment by ramber1836** [comment, marked as answer]
The package of tsfresh has been installed in the pytorch environment.
